client/src/login/LoginContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { LoginForm } from './LoginForm';
+import { authenticateUser } from './LoginActions';
+
+function mapDispatchToProps(dispatch) {
+  return ({
+    onSubmit: (values) => {
+      dispatch(authenticateUser(values.email, values.password));
+    },
+  });
+}
+
+function mapStateToProps() {
+  return ({});
+}
+
+export const LoginContainer = connect(mapStateToProps, mapDispatchToProps)(LoginForm);
```

Comment:
This syntax is typically just used for Higher Order Components. Since this is just a Container component, you would typically create a class here with onSubmit as a class method, and render the LoginForm receiving onSubmit as a prop. This will make it easy to add other Login related components in the render method.